docs/_source/practical_guides/create_dataset.md

```diff
@@ -130,17 +152,17 @@ Once the scope of the project is defined, which implies knowing the `fields`, `q
 
 - `fields`: The list of fields to show in the record card. The order in which the fields will appear in the UI matches the order of this list.
 - `questions`: The list of questions to show in the form. The order in which the questions will appear in the UI matches the order of this list.
-- `guidelines` (optional): A set of guidelines for the annotators. These will appear in the dataset settings in the UI.
 - `metadata`(optional): The list of metadata properties included in this dataset.
 - `extra_metadata_properties` (optional): A boolean to specify whether this dataset will allow metadata fields in the records other than those specified under `metadata`. Note that these will not be accessible from the UI for any user, only retrievable using the Python SDK.
```

Comment:
This should be  `allow_extra_metadata` right?

Reply:
correct